app/views/docs/email-and-sms-templates.phtml

```diff
@@ -0,0 +1,280 @@
+<p>Appwrite uses email and SMS messages to communicate with users to perform authentication actions. Emails and SMS messages can be customized to fit your app's design and voice.</p>
+
+<p>
+    Each Appwrite project can have its own set of unique templates. 
+    Templates also support localization, every template can be written in multiple languages and served 
+    depending on the configured locale.
+</p>
+
+<p>
+<a href="#email-templates"><i class="icon-angle-circled-right"></i>Email Templates</a>
+</p>
+<p>
+<a href="#sms-templates"><i class="icon-angle-circled-right"></i>SMS Templates</a>
+</p>
+
+<h2><a href="#customize" id="customize">Customize Templates</a></h2>
+<p>You can customize email templates for each of your projects in the Appwrite console. </p>
+<div class="notice">
+    <h2>Custom SMTP Server Required</h2>
+    <p>The built-in email service does not support custom email templates. Configure a <a href="#smtp">custom SMTP server</a> to enable custom email templates.</p>
+</div>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>In your project, navigate to the <b>Auth</b> service.</li>
+    <li>Under the <b>Auth</b> service, navigate to the <b>Templates</b> tab.</li>
+    <li>Expand the email template you want to edit.</li>
+    <li>Select the <b>Template language</b>. You can have a different template for each language your app supports.</li>
+    <li>Update the email template fields and click **Update** to save your changes.</li>
+</ol>
+<h2><a href="#email-templates" id="email-templates">Email Templates</a></h2>
+<p>You can customize the email templates for account verification, magic-url authentication, password resets, and user invites.</p>
+<h3><a href="#email-template-components" id="email-template-components">Email Template Components</a></h3>
+<p>Each email templates has the following components that you can customize.</p>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 200px">Component</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>Sender name</td>
+        <td>Readers will see this as display name of the sender.</td>
+    </tr>
+    <tr>
+        <td>Sender email</td>
+        <td>Readers will see this as the displayed email of the sender. This must be a valid email for the SMTP provider you've configured.</td>
```

Comment:
```suggestion
        <td>Readers will see this as a display email of the sender. This email must be authenticated on the SMTP provider you've configured, otherwise it will be delivered to the spam folder. This usually means the email must end with the same domain as your SMTP username.</td>
```